luigi/contrib/hdf5.py

```diff
@@ -0,0 +1,308 @@
+import os
+import luigi
+import logging
+import time
+import sys
+import tempfile
+
+logger = logging.getLogger('luigi-interface')
+
+try:
+    import pandas as pd
+except ImportError:
+    logger.warning("Loading hdf5 module without the python packages pandas. \
+        This will crash at runtime if pandas functionality is used.")
+
+    class Object(): # ensures docs are built will lead to fail at runtime
+        HDFStore = object
+    pd = Object()
+
+if sys.version_info < (3, 3):
```

Comment:
If somebody on StackOverflow or something told you to do this. It's always nice to have a url linking back to that SO-answer as a comment in the source code. :)